index.Rmd

```diff
@@ -1,69 +1,73 @@
 * Data Handling
     * Pandas (Dataframes and Data Analysis)
-    * Numpy (Scientific Computing)
+    * NumPy (Scientific Computing)
 * Visualization
     * Matplotlib (General Graphics)
     * Seaborn (Statistical Graphics)
     * Folium (Interactive Maps)
 * Machine Learning
-    * TensorFlow (Deep Learning)
-    * Keras (Deep/Machine Learning)
-    * Scikit-Learn (Machine Learning)
-
+    * scikit-learn (Machine Learning)
+    * Keras + TensorFlow (Deep Learning)
+    * PyTorch (Deep Learning)
 
-## Python package management with conda
-A set of tools co-exist for installing and managing packages. It is possible to install packages on your base Python interpreter, but sooner or later you will get conflicting Python packages since packages have varying dependencies. It can even [break your system Python interpreter](https://askubuntu.com/questions/95037/what-is-the-best-way-to-install-python-packages). So, instead, we recommend to create a conda environment on your local machine for each project. In conda environments, basically anything, such as software, C libraries or R packages, can be installed, but we use them here for installing Python packages. Packages installed in one environment do not interfere with your base Python or with other conda environments.
+## Python package management with Conda
+A set of tools co-exist for installing and managing Python packages. It is possible to install packages on your base Python interpreter, but sooner or later you will get conflicting Python packages since packages have varying dependencies. It can even [break your system Python interpreter](https://askubuntu.com/questions/95037/what-is-the-best-way-to-install-python-packages).
 
+Instead, we recommend to create a `conda` environment on your local machine for each project. In `conda` environments, basically anything, such as software, C libraries or R packages can be installed. Here we use them here for installing Python packages. Packages installed in one environment do not interfere with your base Python or with other `conda` environments.
 
+## Conda/Mamba installation
+For this course, we will make use of Mamba, fast drop-in reimplementation of the Conda package manager. It has its core parts implemented in C++ for maximum efficiency, makes use of parallel downloading of repository data and package files using multi-threading, and uses `libsolv` for (much) faster dependency solving. 
 
-### Conda installation
-Conda can be downloaded in two versions: Anaconda and Miniconda. The difference is that Miniconda is just shipping the repository management system. So when it is installed, you will only have the management system without packages. Whereas with Anaconda, you get the distribution with most popular Python packages already built in.
+Mamba replaces Conda for most commands, *except for activating and deactivating environments*. You will see more clearly how this works in practice in the first part of this tutorial.
```

Comment:
But we want students to use `source` for activating and deactivating, so this is probably not so useful to mention anyway.

Reply:
OK!